pages/common/vzdump.md

```diff
@@ -0,0 +1,28 @@
+# vzdump
+
+> Backup Utility for virtual machines and containers.
+> More information: <https://pve.proxmox.com/pve-docs/vzdump.1.html>.
+
+- Dump a guest virtual machine into the default dump directory (usually `/var/lib/vz/dump/`), excluding snapshots:
+
+`vzdump {{vm_id}}`
+
+- Backup guest virtual machine with id 101, 102, and 103:
```

Comment:
`Backup` is only the noun, while `back up` is the verb
```suggestion
- Back up the guest virtual machines with the ID 101, 102, and 103:
```

Reply:
`ID` → `IDs` @march